libs/indibase/inditelescope.cpp

```diff
@@ -170,6 +170,28 @@ bool Telescope::initProperties()
         IUFillSwitchVector(&SlewRateSP, SlewRateS, nSlewRate, getDeviceName(), "TELESCOPE_SLEW_RATE", "Slew Rate",
                            MOTION_TAB, IP_RW, ISR_1OFMANY, 0, IPS_IDLE);
 
+    if (CanTrackSatellite())
+    {
+        IUFillText(&TLEtoTrackT[0], "TLE", "TLE", "");
+        IUFillTextVector(&TLEtoTrackTP, TLEtoTrackT, 1, getDeviceName(), "SAT_TLE_TEXT", "Orbit Params", SATELLITE_TAB, 
+        IP_RW, 60, IPS_IDLE);
+
+        char curTime[32] = {0};
+        std::time_t t = std::time(nullptr);
+        struct std::tm *utctimeinfo = std::gmtime(&t);
+        strftime(curTime, sizeof(curTime), "%Y-%m-%dT%H:%M:%S", utctimeinfo);
+        
+        IUFillText(&SatPassWindowT[1], "SAT_PASS_WINDOW_END", "End UTC", curTime);
```

Comment:
use enum instead of 0 and 1 indexes. I think I'll require enums for all new code.

Reply:
That's great! No magic numbers is always better. I created an enum and documented it